tests/workflow_contracts/sccache_test.py

```
@@ -0,0 +1,233 @@
+    build_at = next(
+        (
+            index
+            for index, step in enumerate(steps(definition))
+            if index > zero_at and _is_build_step(step)
+        ),
+        None,
+    )
+    assert build_at is not None and build_at < show_at, (
+        f"{workflow_name}:{job_name} must compile between zeroing its "
+        "counters and reporting them, or the statistics describe no work"
+    )
```

Ruff (0.16.3)

[warning] Assertion should be broken down into multiple parts

(PT018)